src/utils/date.js

```diff
@@ -8,7 +8,11 @@ const TOMORROW = REFERENCE.clone().add(1, 'days').startOf('day')
 const NEXT_WEEK = REFERENCE.clone().add(7, 'days').startOf('day')
 
 export function formatDate (date) {
-  return new Date(date).toLocaleDateString('pt-br')
+  let formatedDate = new Date(date).toLocaleDateString('pt-br')
+  if (formatedDate === '31/12/1') {
```

Comment:
essa string chumbada não sei não, não é melhor usar o moment pra isso ? ele tem um metodo is valid que retorna um boolean

Reply:
hmm, melhor mesmo, vou ver aqui

Reply:
O metodo deles só trata se é valido em tipo, não pra formatar tipo 01/99/2017 ta lgd?

Reply:
mas achei uma maneira melhor mesmo assim